README.md

```diff
@@ -18,6 +18,15 @@ of the pipeline description, and more.
  * **Mailing List**: image-builder@redhat.com
  * **Changelog**: <https://github.com/osbuild/osbuild/releases>
 
+### Principles
+
+1. OSBuild stages are never broken, only deprecated. The same manifest should always produce the same output.
+2. OSBuild stages should be explicit wherever possible instead of e.g. relying on the state of the tree.
+3. OSBuild's pipelines are independent, so the tree is expected to be empty at the beginning of each.
+4. Manifests are expected to be machine-generated, so OSBuild has no convenience functions to support manually created manifests.
+5. OSBuild has confinement around the build environment, but it is not a security feature.
+6. OSBuild needs to run on currently supported RHEL versions.
```

Comment:
Should we extend this also to Fedora and CentOS Stream? That was always the trio we cared about.

Reply:
and rephrase it to "must"?

Reply:
Adjusted for both